ci/docker/runtime_functions.sh

```diff
@@ -1263,17 +1263,17 @@ integrationtest_ubuntu_cpu_onnx() {
-    pytest $COV_ARG --verbose tests/python-pytest/onnx/test_onnxruntime.py::test_obj_class_model_inference_onnxruntime[mobilenetv3_large]
-    pytest $COV_ARG --verbose tests/python-pytest/onnx/test_onnxruntime.py::test_obj_class_model_inference_onnxruntime[resnest200]
-    pytest $COV_ARG --verbose tests/python-pytest/onnx/test_onnxruntime.py::test_obj_class_model_inference_onnxruntime[resnet50_v2]
-    pytest $COV_ARG --verbose tests/python-pytest/onnx/test_onnxruntime.py::test_obj_class_model_inference_onnxruntime[vgg19_bn]
-    pytest $COV_ARG --verbose tests/python-pytest/onnx/test_onnxruntime.py::test_obj_detection_model_inference_onnxruntime[center_net_resnet101_v1b_voc]
-    pytest $COV_ARG --verbose tests/python-pytest/onnx/test_onnxruntime.py::test_img_segmentation_model_inference_onnxruntime[deeplab_resnet50_citys]
-    pytest $COV_ARG --verbose tests/python-pytest/onnx/test_onnxruntime.py::test_pose_estimation_model_inference_onnxruntime[mobile_pose_mobilenet1.0]
-    pytest $COV_ARG --verbose tests/python-pytest/onnx/test_onnxruntime.py::test_action_recognition_model_inference_onnxruntime[inceptionv3_kinetics400]
-    pytest $COV_ARG --verbose tests/python-pytest/onnx/test_onnxruntime.py::test_dynamic_shape_bert_inference_onnxruntime
-    pytest $COV_ARG --verbose tests/python-pytest/onnx/test_onnxruntime.py::test_dynamic_shape_cv_inference_onnxruntime
+    pytest $COV_ARG --verbose tests/python-pytest/onnx/test_onnxruntime_nlp.py::test_bert_inference_onnxruntime
+    pytest $COV_ARG --verbose tests/python-pytest/onnx/test_onnxruntime_cv.py::test_obj_class_model_inference_onnxruntime[mobilenetv3_large]
+    pytest $COV_ARG --verbose tests/python-pytest/onnx/test_onnxruntime_cv.py::test_obj_class_model_inference_onnxruntime[resnest200]
+    pytest $COV_ARG --verbose tests/python-pytest/onnx/test_onnxruntime_cv.py::test_obj_class_model_inference_onnxruntime[resnet50_v2]
+    pytest $COV_ARG --verbose tests/python-pytest/onnx/test_onnxruntime_cv.py::test_obj_class_model_inference_onnxruntime[vgg19_bn]
+    pytest $COV_ARG --verbose tests/python-pytest/onnx/test_onnxruntime_cv.py::test_obj_detection_model_inference_onnxruntime[center_net_resnet101_v1b_voc]
+    pytest $COV_ARG --verbose tests/python-pytest/onnx/test_onnxruntime_cv.py::test_img_segmentation_model_inference_onnxruntime[deeplab_resnet50_citys]
+    pytest $COV_ARG --verbose tests/python-pytest/onnx/test_onnxruntime_cv.py::test_pose_estimation_model_inference_onnxruntime[mobile_pose_mobilenet1.0]
+    pytest $COV_ARG --verbose tests/python-pytest/onnx/test_onnxruntime_cv.py::test_action_recognition_model_inference_onnxruntime[inceptionv3_kinetics400]
+    pytest $COV_ARG --verbose tests/python-pytest/onnx/test_onnxruntime_nlp.py::test_dynamic_shape_bert_inference_onnxruntime
+    pytest $COV_ARG --verbose tests/python-pytest/onnx/test_onnxruntime_cv.py::test_dynamic_shape_cv_inference_onnxruntime
```

Comment:
Why do you need to explicitly list the tests? Alternatively you can choose to mark the tests in the test files and then use the test selection functionality of pytest.

Reply:
Applied mark for pytest group. Please review @szha